Library/Homebrew/build.rb

```diff
@@ -105,7 +105,8 @@ def install
 
     formula.extend(Debrew::Formula) if ARGV.debug?
 
-    formula.brew do
+    formula.brew do |f, stg|
```

Comment:
Is `f` actually used? If not, prefix it with an underscore to make that more obvious (and silence RuboCop).

I think the `stg` is unnecessarily cryptic. What does it stand for? `stage`? What does it actually represent? To me that's a handle to the temporary directory, so something like `temp_directory` (or `staging_directory`, if you prefer) would be nice. (Obviously also applies to the rest of the code.)